src/HonzaBotner.Discord/Command/Permissions/Data/PermissionEntry.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace HonzaBotner.Discord.Command.Permissions.Data
+{
+    public sealed class PermissionEntry
+    {
+        public Guid Id { get; set; }
```

Comment:
Is guid necessary for this, or it just reflection of the database?

Reply:
Do you mean guid as the data type or the overall presence of a unique ID attribute in a DTO object?